src/App.test.tsx

```diff
@@ -1,11 +1,21 @@
 import { render, waitFor } from "@testing-library/react";
-import App from "./App";
+import { Main } from "./pages/Main";
+
+jest.mock("./hooks/useParsedUrlDeskproValues", () => ({
+  useParsedUrlDeskproValues: () => "https://www.google.com",
+}));
+
+jest.mock("./context/deskproDataContext", () => ({
+  useDeskproData: () => ({
+    settings: {},
+  }),
+}));
 
 test("renders App component", async () => {
```

Comment:
does this test check anything?

Reply:
Nope, talked with ash and he said its ok to have no tests on this app